Signal/src/call/PeerConnectionClient.swift

```diff
@@ -392,16 +433,22 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
         // audioTrack is a strong property because we need access to it to mute/unmute, but I was seeing it
         // become nil when it was only a weak property. So we retain it and manually nil the reference here, because
         // we are likely to crash if we retain any peer connection properties when the peerconnection is released
-        Logger.debug("\(TAG) in \(#function)")
-        audioTrack = nil
-        localVideoTrack = nil
-        remoteVideoTrack = nil
+
+        localVideoTrack?.isEnabled = false
+        remoteVideoTrack?.isEnabled = false
+
+        peerConnection.delegate = nil
+        peerConnection.close()
+        peerConnection = nil
```

Comment:
Yes, that's reasonable and that reordering of operations isn't the important part of these changes. I've moved the clearing of the tracks & senders above the clearing of the peerConnection.  But it doesn't seem to make things any safer or less safe.  It's the other changes in this PR that seem to matter in my testing.